src/Evolution/Systems/CurvedScalarWave/Worldtube/AccelerationTermsOrder0.cpp

```diff
@@ -0,0 +1,292 @@
+  DataVector& dv_29 = temps.at(28);
+  dv_29 = sqrt(dv_11);
+  DataVector& dv_30 = temps.at(29);
+  dv_30 = d_16 * dv_9;
+  DataVector& dv_31 = temps.at(13);
+  dv_31 = Dx + d_1 * dv_13 - d_37 * dv_30;
+  DataVector& dv_32 = temps.at(29);
+  dv_32 = Dy + d_1 * dv_7 * yp - d_40 * dv_30;
+  DataVector& dv_33 = temps.at(30);
+  dv_33 = Dx * xpddot + Dy * ypddot;
+  DataVector& dv_34 = temps.at(31);
+  dv_34 = d_2 * dv_6;
+  DataVector& dv_35 = temps.at(32);
+  dv_35 = M * dv_2;
+  DataVector& dv_36 = temps.at(33);
+  dv_36 = d_31 * dv_35;
+  DataVector& dv_37 = temps.at(34);
+  dv_37 = d_32 * dv_7 + d_41 * dv_6 + dv_36;
+  DataVector& dv_38 = temps.at(31);
+  dv_38 = -d_26 * (2. * M * dv_2 * dv_6 * rp + 3. * d_15 * dv_10 * rpdot -
+                   d_15 * dv_9 * rp *
+                       (d_0 * dv_33 + d_6 * dv_34 + dv_34 * rp + dv_37) +
+                   d_20 * d_33 * dv_10 / (d_14 * d_14) - d_28 * dv_3) +
+          dv_31 * xpdot + dv_32 * ypdot;
+  DataVector& dv_39 = temps.at(3);
+  dv_39 = 1.0 / dv_29;
+  DataVector& dv_40 = temps.at(35);
+  dv_40 = dv_14 * dt_fx;
+  DataVector& dv_41 = temps.at(6);
+  dv_41 = d_2 * (-d_4 + d_43 + dv_6);
+  DataVector& dv_42 = temps.at(36);
+  dv_42 = dv_41 * rp;
+  DataVector& dv_43 = temps.at(37);
+  dv_43 = dv_17 * dt_fy;
+  DataVector& dv_44 = temps.at(1);
+  dv_44 =
+      Dx * d_41 +
+      d_2 * (-d_44 * xpdot + dv_1 * xpdot + xp * (d_43 + 2. * dv_4 + dv_5)) -
+      d_34;
+  DataVector& dv_45 = temps.at(5);
+  dv_45 = Dy * d_41 + d_17 * (d_43 + dv_4 + 2. * dv_5) +
+          d_35 * (Dx * ypdot - xpdot * yp) - d_38;
+  DataVector& dv_46 = temps.at(34);
+  dv_46 = dv_9 * (d_0 * (-d_11 - d_12 + dv_33) + d_6 * dv_41 + dv_37 + dv_42);
+  DataVector& dv_47 = temps.at(30);
+  dv_47 = d_9 * dv_2 * ft;
+  DataVector& dv_48 = temps.at(4);
+  dv_48 = 2 * dv_21;
+  DataVector& dv_49 = temps.at(38);
+  dv_49 = 2 * dv_23;
+  DataVector& dv_50 = temps.at(39);
+  dv_50 = 6 * d_7 * dv_24 * rpdot;
+  DataVector& dv_51 = temps.at(40);
+  dv_51 = d_33 * dv_11;
+  DataVector& dv_52 = temps.at(41);
+  dv_52 = d_22 * dv_51;
+  DataVector& dv_53 = temps.at(42);
+  dv_53 = dv_20 * dv_29;
+  DataVector& dv_54 = temps.at(43);
+  dv_54 = dv_18 * dv_29;
+  DataVector& dv_55 = temps.at(44);
+  dv_55 = d_46 * dv_12;
+  DataVector& dv_56 = temps.at(45);
+  dv_56 = d_19 * dv_12;
+  DataVector& dv_57 = temps.at(46);
+  dv_57 = dv_56 * fy;
+  DataVector& dv_58 = temps.at(47);
+  dv_58 = dv_15 * dv_29;
+  DataVector& dv_59 = temps.at(45);
+  dv_59 = dv_56 * fx;
+  DataVector& dv_60 = temps.at(9);
+  dv_60 = dv_29 * dv_9;
+  DataVector& dv_61 = temps.at(48);
+  dv_61 = d_37 * dv_60;
+  DataVector& dv_62 = temps.at(49);
+  dv_62 = dv_31 * dv_39;
+  DataVector& dv_63 = temps.at(50);
+  dv_63 = 6. * d_8 * dv_2 * dv_20;
+  DataVector& dv_64 = temps.at(51);
+  dv_64 = d_46 * dv_29 * dv_7;
+  DataVector& dv_65 = temps.at(16);
+  dv_65 = 3. * dv_16;
+  DataVector& dv_66 = temps.at(18);
+  dv_66 = 3. * dv_19;
```

Comment:
`dv_63`, `dv_65`, and `dv_66` only appear in the combo `dv_63 + dv_65 + dv_66` below. Combine them here instead and only use `dv_66` below
```
dv_66 = dv_63 + dv_65 + 3. * dv_19
```